### PR TITLE
completed MS calculator project

### PR DESCRIPTION
**Pull request by jcast24**
Completed MS calculator project!

### CODE REVIEW

File: jcast24.Calculator/CalculatorLibrary/CalculatorLibrary.cs [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+                    result = num1 - num2;
+                    _writer.WriteValue("Subtract");
+                    break;
+                case "m":
+                    result = num1 * num2;
+                    _writer.WriteValue("Multiply");
+                    break;
+                case "d":
+                    while (num2 != 0)
+                    {
```
Comment (stevietv) [inline jcast24.Calculator/CalculatorLibrary/CalculatorLibrary.cs:90]: Right now any divide doesn't work. It gets stuck in a loop here. Can you fix this up?

**Review by stevietv — CHANGES_REQUESTED** [review]
Hi @jcast24 - thanks for your submission! Looking greate and good work on implementing some extra functions.

I've took a look at it, and before i can approve it there is a bug that needs fixing.

Fix that up and push a new commit so that it gets updated here, once that's done I'll take a look again.

Also, note for the future that its best to have your `sln` file at the top level and not inside one of the project folders.

**Comment by jcast24** [comment]
Fixed the issue with division, thank you for checking over!

File: jcast24.Calculator/CalculatorLibrary/CalculatorLibrary.cs (hunk 1 above)
Comment (jcast24) [reply jcast24.Calculator/CalculatorLibrary/CalculatorLibrary.cs:90]: Fixed the issue, thank you for checking! :)

**Comment by stevietv** [comment]
great job, thanks for updating!

A little tip - the menu for square root shows `s` and not `sr`!
Looking forward to seeing your next work, Approved!

**Pull request closed without merging**